scripts/encode2ga4gh.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+"""
+script to download gene level RNA quantifications from the ENCODE DCC site
+https://www.encodeproject.org and convert to format used by the GA4GH API
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import sys
+import json
+import errno
+import urllib2
+import optparse
+
+import rnaseq2ga
+
+
+class Color(object):
```

Comment:
This seems like a nice utility for many other scripts. Perhaps refactor this code to a generally accessible library in the future?